resources/shaders/rt_hybrid.hlsl

```diff
@@ -262,7 +291,11 @@ void ReflectionHit(inout ReflectionHitInfo payload, in MyAttributes attr)
 	float3 fN = normalize(mul(normal_t, TBN));
 	if (dot(fN, V) <= 0.0f) fN = -fN;
 
-	//TODO: Reflections
+	//Lighting
+	float3 lighting = shade_pixel(hit_pos, V, albedo, metal, roughness, fN, payload.seed, payload.depth);
```

Comment:
This should be removed. This should be handled by the composition pass.

Reply:
How can the composition pass handle recursive reflections? It doesn't have any of the required info.